global.json

```diff
@@ -1,8 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.100-preview.3.21202.5",
-    "allowPrerelease": true,
-    "rollForward": "major"
```

Comment:
📝 Not what I set out to do, but required for build to succeed. We can investigate changing this setting later.

Reply:
As many times as we've changed this I'm wondering what the "best" answer is here that we should be striving towards. May want to sync up with sdk team later on and explain the issues we've been having here. Perhaps we're missing an easy answer here.